core/controllers/admin.py

```diff
@@ -631,3 +632,23 @@ def get(self):
             'data': extracted_answers
         }
         self.render_json(response)
+
+
+class SendDummyMailToAdminHandler(base.BaseHandler):
+    """This function handles sending test emails."""
+
+    @acl_decorators.can_access_admin_page
+    def post(self):
+        username = self.payload.get('username')
+        if feconf.CAN_SEND_EMAILS:
+            response = {
+                'msg': 'Success! Mail sent to admin.',
+            }
+            dummy_email_name = feconf.SYSTEM_EMAIL_NAME
+            dummy_email_address = feconf.SYSTEM_EMAIL_ADDRESS
+            email_body = 'This is a test mail from ' + username
```

Comment:
We use string interpolation in this codebase: `'This is a test mail from %s.' % username`

Reply:
Done.